cadquery/assembly.py

```diff
@@ -619,6 +628,30 @@ def shapes(self) -> List[Shape]:
 
         return rv
 
+    @property
```

Comment:
This (all props) should be removed. I.e. the idea was to make those fields private

Reply:
I removed those properties, although I accidentally made the commit message "Added...", which is the opposite of what I did.